<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterceptGradientVolumeDialog</class>
 <widget class="QDialog" name="InterceptGradientVolumeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>426</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Intercept and Gradient Volumes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Dataset:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QComboBox" name="cbDataset"/>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="cbSuperGather">
        <property name="text">
         <string>Use Supergathers</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Inlines to Combine:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2" colspan="2">
       <widget class="QSpinBox" name="sbInlines">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Crosslines to Combine:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QSpinBox" name="sbCrosslines">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>9</number>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QCheckBox" name="cbRestrictOffset">
        <property name="text">
         <string>Restrict Offsets</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Maximum Absolute Offset:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4">
       <widget class="QLineEdit" name="leMaxOffset">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>1000</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QCheckBox" name="cbTimeWindow">
        <property name="text">
         <string>Use a Time Window</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Window Start [Milliseconds]:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="4">
       <widget class="QLineEdit" name="leWindowStart">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Window End [Milliseconds]:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="4">
       <widget class="QLineEdit" name="leWindowEnd">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Intercept:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leIntercept"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Gradient:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leGradient"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InterceptGradientVolumeDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InterceptGradientVolumeDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbRestrictOffset</sender>
   <signal>toggled(bool)</signal>
   <receiver>leMaxOffset</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>167</x>
     <y>160</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSuperGather</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbInlines</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbSuperGather</sender>
   <signal>toggled(bool)</signal>
   <receiver>sbCrosslines</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>71</x>
     <y>178</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTimeWindow</sender>
   <signal>toggled(bool)</signal>
   <receiver>leWindowStart</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>240</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbTimeWindow</sender>
   <signal>toggled(bool)</signal>
   <receiver>leWindowEnd</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>213</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
